website/docs/r/kinesis_firehose_delivery_stream.html.markdown

```diff
@@ -410,7 +410,7 @@ The `s3_configuration` object supports the following:
 * `buffer_size` - (Optional) Buffer incoming data to the specified size, in MBs, before delivering it to the destination. The default value is 5.
                                 We recommend setting SizeInMBs to a value greater than the amount of data you typically ingest into the delivery stream in 10 seconds. For example, if you typically ingest data at 1 MB/sec set SizeInMBs to be 10 MB or higher.
 * `buffer_interval` - (Optional) Buffer incoming data for the specified period of time, in seconds, before delivering it to the destination. The default value is 300.
-* `compression_format` - (Optional) The compression format. If no value is specified, the default is UNCOMPRESSED. Other supported values are GZIP, ZIP & Snappy. If the destination is redshift you cannot use ZIP or Snappy.
+* `compression_format` - (Optional) The compression format. If no value is specified, the default is `UNCOMPRESSED`. Other supported values are `GZIP`, `ZIP`, `Snappy`, & `HADOOP_SNAPPY`. If the destination is redshift you cannot use `ZIP`, `Snappy`, or `HADOOP_SNAPPY`.
```

Comment:
Just wanted to double-check: AWS api docs say only `Snappy` and `ZIP` are not supported when using redshift (seen here: https://docs.aws.amazon.com/firehose/latest/APIReference/API_S3DestinationConfiguration.html#Firehose-Type-S3DestinationConfiguration-CompressionFormat) , but that could be incorrect on their part..have you found that `HADOOP_SNAPPY` is also not supported?

Reply:
Thanks for asking. I should maybe remove this given the doc you linked differs, however when I used the AWS console to create a new stream targeting my Redshift cluster the only compression option I could pick was GZIP.

Reply:
HI @bnix, apologies for the delay! On my end, I've tested using all the supported `compression_format` values with a `redshift` destination, and strangely the configurations apply successfully without error as suggested by the AWS API docs (it's possible the SDK supports values not visible in the console 🙃 ). I'll update this accordingly (if the git branch is writeable :) )